src/core/trulens/core/database/migrations/data.py

```diff
@@ -16,41 +16,36 @@
 from trulens.core.schema import record as mod_record_schema
 from trulens.core.utils.pyschema import FunctionOrMethod
 
-sql_alchemy_migration_versions: List[str] = ["1"]
-"""DB versions that need data migration.
+sql_alchemy_migration_versions: List[int] = [1]
```

Comment:
Does the typing play a role? Our migrations are ascending integers but are typed as strings and can be arbitrary strings

Reply:
Yes, but the code was actually assuming they were all ints anyway so I just made it more explicit.